### PR TITLE
pybind/test_volume_client: print python version correctly

### PR DESCRIPTION
**Pull request by LianneWang**
Fixes: http://tracker.ceph.com/issues/40184

Signed-off-by: Lianne <liyan.wang@xtaotech.com>

### CODE REVIEW

**Comment by LianneWang** [comment]
@batrick  make check failed , there is nothing about my commit,  help check please.

**Comment by tchaikov** [comment]
retest this please.

**Comment by LianneWang** [comment]
> retest this please.

@tchaikov @smithfarm  "Unmodified Submodules"  failed , check the cause of the error please

**Comment by LianneWang** [comment]
@tchaikov @smithfarm "make check (arm64)" failed again, Solve the problem please

File: qa/tasks/cephfs/test_volume_client.py [hunk 1]
```diff
@@ -19,7 +19,8 @@ class TestVolumeClient(CephFSTestCase):
     def setUp(self):
         CephFSTestCase.setUp(self)
         self.py_version = self.ctx.config.get('overrides', {}).get('python', 'python')
-        log.info("using python version: %s".format(self.py_version))
+        log.info("using python version: {python_version}".format(
```
Comment (tchaikov) [inline qa/tasks/cephfs/test_volume_client.py]:
```suggestion
        log.info("using python version: %r", self.py_version)
```

**Comment by LianneWang** [comment]
@tchaikov thanks, but I think my code is ok, and I still feel that there is a problem with "make check".
 There has the same problem:http://tracker.ceph.com/issues/36405

**Comment by tchaikov** [comment]
@LianneWang please see https://docs.python.org/2/library/logging.html#logging.Logger.debug . it's perfectly fine if you insist, i just wanted to remind that the logging module is supposed to use that way.

yeah, i am aware of this issue. @cyx1231st is looking into it.

**Comment by batrick** [comment]
@LianneWang normally we just use "qa: " for the prefix but this is okay.

File: qa/tasks/cephfs/test_volume_client.py (hunk 1 above)
Comment (rishabh-d-dave) [reply qa/tasks/cephfs/test_volume_client.py]: just curious: is the suggested only for having narrower lines/conciseness or are there more advantages to it?

**Comment by LianneWang** [comment]
@batrick  thanks，already changed,  please check again

**Comment by batrick** [comment]
@LianneWang i think you linked to the wrong ticket. Ticket 22340 is for your other PR #19371.

**Comment by LianneWang** [comment]
@batrick sorry about that, update again

**Comment by batrick** [comment]
@LianneWang I don't see the update yet. Did you forget to push?

**Comment by LianneWang** [comment]
@batrick  update again,  please check again

**Pull request merged**